### ISSUE TITLE
New title?

### ISSUE THREAD

**Issue by tomcrick** [tomcrick/FMFun2019#1]
Mentioned by both R1:

_The title does not match with the submission. From the title, I would expect to see what’s the problems of education in formal method and cybersecurity and how to solve the problem. But the submission put the most effort on providing some researches in the cybersecurity domain, and has only a few sentences about education in the very late part. It only provides some surface problems of education such as lack of initiative，lack of practice and lack of educational resources. It could be interesting to see how to solves these problems._

...and  and R3:

_The title and keywords are misleading since the content is not about education at student level. Education in a wider sense is addressed, which incorporates: change in people's attitude in use and acceptance of formal methods, use of coding methods, tool support, managers, designers and last but not least student education._

_Title, keywords and content should be revised, and made clear what the content is about and what the main claim and goals are._  

_To me the title is a bit misleading. In the paper I can find the claim that formal methods can help to improve, but there are other important points that are made, such as coding approaches, tools, education, ... that need to change and/or improve. To me 'Improving Cybersecurity Education', sounds more applicable. I would also leave out the keywords 'undergraduate' and 'curricula'. Also 'UK' seems not important to me._

**Comment by JamesHDavenport** [comment]
Good point. The paper was submitted to FMFun, so te title makes sense in that context. But probably not i a braider context.  "CyberSecurity as a motivation for Formal Methods"??
